src/appengine/handlers/cron/bandit_probabilities.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Bandit probabilities cron job. Queries from bigquery for updated probability
+	values based on new edges for various combined strategies."""
+
+from datastore import data_types
+from datastore import ndb
+from google_cloud_utils import big_query
+from handlers import base_handler
+from libs import handler
+
+MULTIPLE_STRATEGY_PROBABILITY_QUERY = """
+SELECT 
+  EXP(standardized_avg_ne / temp_p) / (SUM(EXP(standardized_avg_ne / temp_p)) OVER()) as edges_bandit,
+  EXP(standardized_avg_cc / temp_p) / (SUM(EXP(standardized_avg_cc / temp_p)) OVER()) as crash_bandit,
+  EXP(standardized_avg_nf / temp_p) / (SUM(EXP(standardized_avg_nf / temp_p)) OVER()) as features_bandit,
+  EXP(standardized_avg_nu / temp_p) / (SUM(EXP(standardized_avg_nu / temp_p)) OVER()) as units_bandit,
+  strategy,
+  strategy_count 
+FROM
+  (SELECT
+	CONCAT(s_radamsa, s_max_len, s_ml_rnn, s_vp, s_corpus, s_fork, s_subset, s_recommended_dict) AS strategy,
+	AVG((crash_count - avg_cc) / stddev_cc) as standardized_avg_cc,
+	AVG((new_edges - avg_ne) / stddev_ne) as standardized_avg_ne,
+	AVG((new_features - avg_nf) / stddev_nf) as standardized_avg_nf,
+	AVG((new_units_added - avg_nu) / stddev_nu) as standardized_avg_nu,
+	COUNT(*) as strategy_count,
+	"any" as matcher,
+	/*CHANGE TEMPERATURE HERE*/
```

Comment:
Treat these the same as other comments, start with a capital and use normal punctuation before the end.

Nit for these specifically: spaces after initial * and before final *